### DISCUSSION TITLE
using manim without nvidia

### DISCUSSION THREAD

**Discussion by tuple0110** [3b1b/manim#1621] [Q&A]
My new laptop doesn't have NVIDIA graphics card in it, and any line or Mobjects containing line are not displayed. Is there any way to use Manim without NVIDIA graphics card?
(I have intel iris integrated graphics card)

**Comment by jesi-rgb** [comment]
That's weird, but I don't think it is a problem of lacking NVIDIA graphics. I am no expert, but I am currently running ManimGL in an Apple M1 Mac Mini, which does not have NVIDIA graphics by any means, since everything in there is Apple based, not even Intel.

So probably the problem with lines lies behind some other reasons. Can you provide some examples?
